ddtrace/contrib/pytest/newhooks.py

```diff
@@ -0,0 +1,29 @@
+"""pytest-ddtrace hooks.
+
+These hooks are used to provide extra data used by the Datadog CI Visibility plugin.
+
+For example: module, suite, and test names for a given item.
+
+Note that these names will affect th display and reporting of tests in the Datadog UI, as well as information stored
+the Intelligent Test Runner. Differing hook implementations may impact the behavior of Datadog CI Visibility products.
+"""
+
+import pytest
+
+
+@pytest.hookspec(firstresult=True)
+def pytest_ddtrace_get_item_module_name(item):
+    # type (Pytest.item) -> str
```

Comment:
Making these type comments into Py3 annotations would increase the library's interoperability with type-checking tools and reduce the risk of type-related confusion when working with this code.